components/local-app/main.go

```diff
@@ -92,11 +106,49 @@ func run(host, sshConfig string) error {
 		Context: context.Background(),
 		Token:   tkn,
 		Log:     logrus.NewEntry(logrus.New()),
+		ReconnectionHandler: func() {
+			if b != nil {
+				b.FullUpdate()
+			}
+		},
 	})
 	if err != nil {
 		return err
 	}
-	b := bastion.New(client, cb)
+
+	b = bastion.New(client, cb)
+	go http.ListenAndServe("localhost:5000", handlers.CORS(
```

Comment:
This looks like the beginning of the local app API, no?
Is this a left-over from previous testing (i.e. can be removed), or would it help if that stuck around until we have the API?

Reply:
I use it to play with VS Code web integration. We can turn it in proper grcp web endpoint in next PR.